tests/st/utils/utils.py

```diff
@@ -409,6 +409,18 @@ def wipe_etcd(ip):
     curl_etcd("calico/v1/config/UsageReportingEnabled",
                    options=["-XPUT -d value=False"], ip=ip)
 
+    etcd_container_name = "calico-etcd"
+    tls_vars = ""
+    if ETCD_SCHEME == "https":
```

Comment:
seems like you and @lwr20's PR (https://github.com/projectcalico/calicoctl/pull/1784) both have this change and Makefile changes in common, I'm guessing you'll sort it out?

Reply:
I just pushed an update removing the commented include so hopefully this passes so this can be merged and then @lwr20 can rebase (or I will if his makes it in first), I don't think we have anything that conflicts so I don't think it matters much.